docs/03-tutorials/00-serverless/svls-12-use-opentelemetry-client.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Use the OpenTelemetry standard
+---
+
+This tutorial shows how to use the build-in OpenTelemetry tracer object to send custom trace data to the Jaeger service.
+
+Kyma Functions are instrumented to handle trace headers. This means that every time you call your Function, the executed logic is traceable using a dedicated span visible in the Jaeger service (i.e. start time and duration).
+Additionally, you can extend the default trace context and create your own custom spans whenever you feel it is helpful (i.e. when calling a remote service in your distributed application) or add additional information to the tracing context by introducing events and tags. This tutorial shows you how to do it using tracer client that is available as part of the [event](../../05-technical-reference/svls-08-function-specification.md#event-object) object.
+
+## Prerequisites
+
+Before you start, make sure you have these tools installed:
+
+- Kyma installed on a cluster
+
+## Steps
+
+Follows these steps:
+
+1. [Create inline Function](./svls-01-create-inline-function.md) with the following body:
+
+   <div tabs name="code" group="functions-code">
+   <details>
+   <summary label="node.js">
+   Node.js
+   </summary>
+
+      ```javascript
+      module.exports = {
+         main: function (event, context) {
+            span = event.tracer.startSpan('foo');
+            span.addEvent('bar');
+            span.end();
+            
+            return "hello OpenTelemetry"
+         }
+      }
+      ```
+
+   </details>
+   <details>
+   <summary label="python">
+   Python
+   </summary>
+
+      ```python
+      def main(event, context):
+         span = event.tracer.start_span("foo")
+         span.add_event("bar")
+         span.end()
+
+         return "hello OpenTelemetry"
+      ```
+
+   </details>
+   </div>
+
+2. [Expose your Function](./svls-03-expose-function.md) and access the Function's external address.
```

Comment:
```suggestion
2. [Expose your Function](./svls-03-expose-function.md).
```
I guess we do not need to rest of the sentence as everything is described under the link.